tests/vectors/main.rs

```diff
@@ -70,10 +45,12 @@ mod profile_snapshot_tests {
 
     #[test]
     fn v100_100() {
-        let (profile, json) = fixture_and_json::<Profile>(
-            "only_plaintext_profile_snapshot_version_100",
-        )
-        .expect("V100 Profile to deserialize");
+        let (profile, json) =
+            fixture_and_json::<Profile>(include_str!(concat!(
```

Comment:
seemingly small change, alas, quite big in fact, a great improvement, since these paths are COMPILE TIME guaranteed! my earlier solution was not compile time guarantees. `include_str!` is an amazing macro, giving compile time guarantees :)